### PR TITLE
chore: improve coverage for callable membrane refactor

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by rwaldron** [comment]
Compare to https://307-206912788-gh.circle-artifacts.com/0/coverage-merged/index.html

**Comment by rwaldron** [comment]
Improvements: https://315-206912788-gh.circle-artifacts.com/0/coverage-merged/index.html

**Comment by rwaldron** [comment]
https://317-206912788-gh.circle-artifacts.com/0/coverage-merged/index.html

File: packages/near-membrane-base/src/membrane.ts [hunk 1]
```diff
@@ -1556,3 +1556,5 @@ export function init(
         apply(callableLinkUnforgeables, undefined, unforgeablePointers);
     };
 }
+
+export const initSourceTextInStrictMode = `(function(){'use strict';return (${init.toString()})})()`;
```
Comment (caridy) [inline packages/near-membrane-base/src/membrane.ts:1560]: this is a great idea.

File: packages/near-membrane-base/src/__tests__/environment.spec.ts [hunk 2]
```diff
@@ -152,14 +153,279 @@ describe('VirtualEnvironment', () => {
...
+
+            const redValue = {};
+            const endowments = {
+                0: 'foo',
+            };
+
+            ve.lazyRemap(redValue, Object.keys(getFilteredEndowmentDescriptors(endowments)));
+            expect(Object.getOwnPropertyNames(redValue).length).toBe(0);
+        });
+
```
Comment (jdalton) [inline packages/near-membrane-base/src/__tests__/environment.spec.ts:191]: ☝️test `'Skips index-like properties'` looks to be covered by `'Skips index-like properties, called after remap'` 👇

**Review by jdalton — APPROVED** [review]

**Pull request merged**